cgmanifests/submodules/cgmanifest.json

```diff
@@ -5,9 +5,9 @@
       "Component": {
         "Type": "other",
         "other": {
-          "Name": "patchelf",
-          "Version": "0.12",
-          "DownloadUrl": "https://github.com/NixOS/patchelf/archive/0.12.tar.gz"
+          "Name": "autoconf",
```

Comment:
nit: this cgmanifests.json file was originally intended for git submodules only. this is why it is in the "submodules" subdirectory. maybe we can clean it up later (make a new file or rename things).